### DISCUSSION TITLE
The user specified as a definer ('mysql.infoschema'@'localhost') does not exist

### DISCUSSION THREAD

**Discussion by phpconsult** [Homebrew/discussions#79] [Everyday usage]
I installed mysql on OSX via Homebrew. However, when I try to create a table or even after entering a command like "show databases" I get the error message as shown in the title. I did a re-install already, but to no avail. Can anyone help me out  here?

PS user infoschema dóes exit.

**Comment by SMillerDev** [comment]
What commands are you running to install mysql? What commands are you running to enter a command?

**Reply by phpconsult** [reply]
brew install mysql

unset TMPDIR

mkdir -p ~/Library/LaunchAgents

cp /usr/local/Cellar/mysql/8.0.21_1/homebrew.mxcl.mysql.plist ~/Library/LaunchAgents/

launchctl load -w ~/Library/LaunchAgents/homebrew.mxcl.mysql.plist
